src/content/docs/more-integrations/terraform/terraform-intro.mdx

```diff
@@ -64,7 +64,7 @@ With `terraform` all set, configure the New Relic `provider` with the following
 
 1. Your [New Relic Account ID](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/account-id).
 2. Your New Relic <InlinePopover type="userKey" />. Most user keys begin with the prefix `NRAK-`.
-3. Your [New Relic region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account). Your region is `US` if your account settings page is located at `one.newrelic.com`, and `EU` if your account is located at `one.eu.newrelic.com`.
+3. Your [New Relic region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account). Your region is `US` if your account settings page is located at one.newrelic.com, and `EU` if your account is located at one.eu.newrelic.com.
```

Comment:
@paperclypse What do you think about removing the formatting from the URL as well the region acronym (EU, US, etc.)?

Reply:
How did this terraform doc get into my PR? Did just accidentally drag some other changes into it? @ally-sassman

Reply:
No, that was my mistake! I removed those edits